README.md

```diff
@@ -17,19 +17,19 @@
 [npm-version]: https://img.shields.io/npm/v/@dtinsight/molecule.svg?style=flat-square
 [npm-version-url]: https://www.npmjs.com/package/@dtinsight/molecule
 
-[中文](./README-zhCN.md) | [English](./README.md)
+[中文](./README-zhCN.md) | [English](./README.md) | [한국어](./README-koKR.md)
 
 The **Molecule** is a lightweight **Web IDE UI** framework which is built with React.js and inspired by the VSCode. We have designed the Extension APIs similar to the VSCode, to help developers extend the Workbench in an easier way. It's convenient to integrate the Molecule with React.js applications. It has been applied to many products in [DTStack](https://www.dtstack.com/)
 
 [Online Preview](https://dtstack.github.io/molecule-examples/#/)
 
 ## Features
 
--   Built-in the VSCode **Workbench** UI
--   Compatible with the VSCode **ColorTheme**
+-   Built-in the Visual Studio Code **Workbench** UI
+-   Compatible with the Visual Studio Code **ColorTheme**
 -   Customize the Workbench via **React Component** easily
 -   Built-in Monaco-Editor **Command Palette, Keybinding** features
--   Support the **i18n**, built-in zhCN, and English
+-   Support the **i18n**, built-in Simplified Chinese, Korean and English
```

Comment:
Sync these updates to files:

```bash
website/docs/introduction
website/docs/guides/extend-locales
website/i18n/zh-CN/current/introduction
website/i18n/zh-CN/current/guides/extend-locales
```

Reply:
I'm sorry, but do you plan to keep the documentation site in Chinese?
https://dtstack.github.io/molecule/en/docs